include/data_management/data/internal/conversion.h

```diff
@@ -48,12 +48,19 @@ inline ConversionDataType getConversionDataType<double>()  { return DAAL_DOUBLE;
 template<>
 inline ConversionDataType getConversionDataType<float>()   { return DAAL_SINGLE; }
 
+typedef void (*vectorConvertFuncType)(size_t n, const void * src, void * dst);
+typedef void (*vectorStrideConvertFuncType)(size_t n, const void * src, size_t srcByteStride, void * dst, size_t dstByteStride);
 
-typedef void(*vectorConvertFuncType)(size_t n, const void *src,
-                                               void *dst);
+#if defined(__INTEL_COMPILER)
+typedef bool(*vectorCopy2vFuncType)(const size_t nrows, const size_t ncols, void* dst, void* ptrMin, DAAL_INT64* arrOffests);
```

Comment:
Last parameter. Did you mean 

> `arrOffsets`
 instead of 
>  `arrOffests `

Reply:
done